README.md

```diff
@@ -5,6 +5,8 @@ cf\_units
 
 \(C) British Crown Copyright 2015 - 2016, Met Office
 
+Introduction
+------------
```

Comment:
This is an RST style heading, markdown headings use the `#`, see https://help.github.com/articles/basic-writing-and-formatting-syntax/#headings